### DISCUSSION TITLE
max music player

### DISCUSSION THREAD

**Discussion by mmtxrc** [z-huang/music#620] [General]
i saw [this app](https://play.google.com/store/apps/details?id=com.zionhuang.music) listed on the play store.. is this a dev or a contributor here? is it a scam? no credits or mention of this project.
is it okay for any person to take the src code of a foss project and compile and publish it anywhere? i wonder if thats allowed in the license used here. i found it fishy.

**Comment by kach17** [comment]
Innertune is licensed under GPL so commercial distribution is allowed I think. Although it feels bad to see it being published on playstore without credits. Moreover that guy seemingly didn't even bother to change anything

**Reply by mmtxrc** [reply]
not a speck

**Reply by TheTrashCrafter** [reply]
Well, it's modified. So the Source Code has to be available for everyone

**Reply by selurvedu** [reply]
He at least had to change the app ID. He did not.

**Reply by mmtxrc** [reply]
he cud not. he dum
